src/main/resources/assets/epicfightnbt/epicfightnbt.json

```diff
@@ -0,0 +1,134 @@
+{
+  "weapons": [
```

Comment:
Can't we simplify this to be in the form
```
{
    "double/head_left": {
        "subtypes": [
            {
                "armor_ignorance": 20,
                "hit_at_once": 0,
                "impact": 4,
                "weapon_type": "axe",
                "nbt_value": "double/basic_hammer_left"
            },
            ...
        ]
    },
    ...
}
```
In `NBTCategorySchema` would GSON be able to deserialize a map into this form automatically? If not that's fine, this is a minor thing.

Reply:
Working on it now